app/src/main/java/org/thoughtcrime/securesms/groups/compose/CreateGroupScreen.kt

```diff
@@ -0,0 +1,183 @@
+        showLoading = viewModel.isLoading.collectAsState().value,
+        items = viewModel.selectContactsViewModel.contacts.collectAsState().value,
+        onCreateClicked = viewModel::onCreateClicked,
+        onBack = onBack,
+    )
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun CreateGroup(
+    groupName: String,
+    onGroupNameChanged: (String) -> Unit,
+    groupNameError: String,
+    contactSearchQuery: String,
+    onContactSearchQueryChanged: (String) -> Unit,
+    onContactItemClicked: (accountID: AccountId) -> Unit,
+    showLoading: Boolean,
+    items: List<ContactItem>,
+    onCreateClicked: () -> Unit,
+    onBack: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    val focusManager = LocalFocusManager.current
+
+    Scaffold(
+        containerColor = LocalColors.current.backgroundSecondary,
+        topBar = {
+            BackAppBar(
+                title = stringResource(id = R.string.groupCreate),
+                backgroundColor = LocalColors.current.backgroundSecondary,
+                onBack = onBack,
+            )
+        }
+    ) { paddings ->
+        Box(modifier = modifier.padding(paddings),) {
+            Column(
+                modifier = modifier.padding(vertical = LocalDimensions.current.spacing),
+                horizontalAlignment = Alignment.CenterHorizontally,
+                verticalArrangement = Arrangement.spacedBy(LocalDimensions.current.spacing)
+            ) {
+                SessionOutlinedTextField(
+                    text = groupName,
+                    onChange = onGroupNameChanged,
+                    placeholder = stringResource(R.string.groupNameEnter),
+                    textStyle = LocalType.current.base,
+                    modifier = Modifier.padding(horizontal = LocalDimensions.current.spacing),
+                    error = groupNameError.takeIf { it.isNotBlank() },
+                    enabled = !showLoading,
+                    innerPadding = PaddingValues(LocalDimensions.current.smallSpacing),
+                    onContinue = focusManager::clearFocus
+                )
+
+                SearchBar(
+                    query = contactSearchQuery,
+                    onValueChanged = onContactSearchQueryChanged,
+                    placeholder = stringResource(R.string.searchContacts),
+                    modifier = Modifier.padding(horizontal = LocalDimensions.current.spacing),
+                    enabled = !showLoading
+                )
+
+                LazyColumn(modifier = Modifier.weight(1f)) {
+                    multiSelectMemberList(
+                        contacts = items,
+                        onContactItemClicked = onContactItemClicked,
+                        enabled = !showLoading
+                    )
+                }
+
+                PrimaryOutlineButton(
+                    onClick = onCreateClicked,
+                    modifier = Modifier.widthIn(min = 120.dp)
```

Comment:
Where is this 120 coming from? Figma seem to show 160 for these, and if this is a common thing we should promote the 160.dp value to the LocalDimensions for re-use